docs/MONGO_DB_SETUP.md

```diff
@@ -0,0 +1,27 @@
+# Mongo_db setup
+1. first thing you need to go to this url [mongo_db login](https://account.mongodb.com/account/login) directly.
```

Comment:
Points should start with caps ( Capital letters )

Also there are some grammatical errors please solve them. 
@amir-ammar